spec/features/user_can_unlike_post_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.feature "Unlike", type: :feature do
+  scenario "Can unlike a post" do
+    sign_up(email: "test@test.com", password: "password")
+    visit "/posts"
+    click_link "New post"
+    fill_in "Message", with: "Hello, world!"
+    click_button "Submit"
+
+    expect(page).to have_content(:thumbsup)
+
```

Comment:
Line 11 was already tested in 'can see the like emoji next to a post'. Is it needed again here?

Similarly for lines 15 and 16 - already tested elsewhere - may be good to delete these doubled up expect lines to be more DRY?